src/main/java/reposense/git/GitBlame.java

```diff
@@ -81,7 +81,7 @@ private static GitBlameLineInfo processGitBlameResultLine(String blameResult) {
         String commitHash = blameResultLines[0].substring(0, FULL_COMMIT_HASH_LENGTH);
         String authorName = blameResultLines[1].substring(AUTHOR_NAME_OFFSET);
         String authorEmail = blameResultLines[2].substring(AUTHOR_EMAIL_OFFSET).replaceAll("[<>]", "");
-        long timestampMilliseconds = Long.parseLong(blameResultLines[5].substring(COMMIT_TIME_OFFSET));
+        long timestampMilliseconds = Long.parseLong(blameResultLines[3].substring(AUTHOR_TIME_OFFSET));
```

Comment:
It seems like `author-time` is semantically what a user would expect from the diff. We can offer options of `AUTHOR_TIME_OFFSET` or `COMITTER_TIME_OFFSET` if necessary, but probably not worth it. Let's look at the tests this affects and see if we can update them.

Reply:
Hi, after some investigation, this indeed caused one of the test cases to fail, AuthorAnalyzerTest > analyzeAuthorship_beforeSinceDate_success(). I have added an option to use either author time or committer time in this method.